README.md

```diff
@@ -29,12 +32,14 @@ If the file is present, the file will be copied and the original will be moved t
 
 ## Prerequisites
 
+## Software and services
+
 - Node.js LTS 16
 - access to an Azure blob storage account (see options below)
 - access to Azure file share storage account
 - [Azure Functions Core Tools V3](https://docs.microsoft.com/en-us/azure/azure-functions/functions-run-local?tabs=v4%2Clinux%2Ccsharp%2Cportal%2Cbash)
 
```

Comment:
Missing Configuration header

Reply:
Oh no, wait it's not -- remove this section on Azure Blob Storage

Add a configuration Azure Blob Storage section copying from `batch-processor`

It pushes files from Azure Blob Storage to a `dax` container.

Reply:
There are no automatically created directories